code/modules/reagents/reagent_containers/dropper.dm

```diff
@@ -5,7 +5,7 @@
 	icon_state = "dropper0"
 	inhand_icon_state = "dropper"
 	worn_icon_state = "pen"
-	amount_per_transfer_from_this = 5
```

Comment:
how was this broken?

Reply:
State and transfer amount needs to be the same, aka 1,1 or 5,5.
two ways of fixing it, add the missing variable that changes state to 5, or simply change the amount(spawned with) to match the default made in reagent_containers.